pkg/apis/pipeline/v1/taskrun_types.go

```diff
@@ -338,9 +338,10 @@ func (trs *TaskRunStatus) SetCondition(newCond *apis.Condition) {
 // StepState reports the results of running a step in a Task.
 type StepState struct {
 	corev1.ContainerState `json:",inline"`
-	Name                  string `json:"name,omitempty"`
-	Container             string `json:"container,omitempty"`
-	ImageID               string `json:"imageID,omitempty"`
+	Name                  string          `json:"name,omitempty"`
+	Container             string          `json:"container,omitempty"`
+	ImageID               string          `json:"imageID,omitempty"`
+	Results               []TaskRunResult `json:"results,omitempty"`
```

Comment:
We renamed `PipelineRunResult` and `TaskRunResult` to `Result` in v1 (probably for duck typing?).

I like the name `Result` since it is under `StepStatus`. Note that We call it  `Result` for `TaskRun results` as well.

Reply:
Alias sounds good though.

Reply:
I created an alias `StepRunResult`.